src/components/frame/SubscribedColoniesList/SubscribedColoniesList.tsx

```diff
@@ -49,7 +51,27 @@ const SubscribedColoniesList = () => {
             <SpinnerLoader appearance={{ size: 'medium' }} />
           </div>
         )}
-        {!isMobile &&
+        {!userLoading && isMobile ? (
+          <>
+            {watchlist.length ? (
+              <SubscribedColoniesDropdown coloniesList={watchlist} />
```

Comment:
I think there's a problem with this prop. It's expected type is `WatchedColonies[]`, but `watchlist` comes from `user.watchlist` which we query in the following way:
```gql
watchlist {
    items {
      colony {
        ...WatchedColony
      }
      createdAt
    }
  }
```
So the items are not of type `WatchedColonies[]`. Since we're not using a fragment to query them, they don't have a specific type generated and their type is:
```js
items: {
  colony: WatchedColonyFragment;
  createdAt: string;
}
```

`WatchedColonies` refer to this model in the API:
<img width="292" alt="image" src="https://user-images.githubusercontent.com/112586815/209239307-cb2d03c0-4726-471b-b6ac-c8d7261764be.png">
In most cases, the types generated from GraphQL models will differ to what we're using in the CDapp - as we can choose to only select a subset of fields rather than all of them. We can also rename fields, which influences the type.

So, if you need the type of items, I'd suggest you create a fragment for `items` which will generate the type for you. This should also fix the type errors in `SubscribedColoniesDropdown`:

<img width="521" alt="image" src="https://user-images.githubusercontent.com/112586815/209239621-a7f845e0-4c68-4119-bab7-57e7fe0a43c9.png">

Feel free to message me if you have any questions